java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -1209,6 +1215,137 @@ public static CompletableFuture<Object> toggle(
                 client, new ArgsBuilder().add(gs(JSON_TOGGLE)).add(key).add(path).toArray());
     }
 
+    /**
+     * Retrieve the JSON document stored at <code>key</code>. The returning result is in the Valkey or Redis OSS Serialization Protocol (RESP).
```

Comment:
nit:
```suggestion
     * Retrieves the JSON document stored at <code>key</code>. The returning result is in the Valkey or Redis OSS Serialization Protocol (RESP).
```

Reply:
also you should probably mention that the path is defaulted to the root for the signatures without `path`

Reply:
`the JSON document` indicates the same, as that's what you'll get with a root path.